src/frontend/assets/js/ndt-browser-client.js

```diff
@@ -529,7 +529,7 @@ NDTjs.prototype.ndtMetaTest = function(ndtSocket) {
       that.callbacks.onstatechange('running_meta', that.results);
       // Send one piece of meta data and then an empty meta data packet
       ndtSocket.send(
-        that.makeNdtMessage(that.TEST_MSG, 'client.os.name:NDTjs'),
+        that.makeNdtMessage(that.TEST_MSG, 'client.os.name:NDTjsPiecewise'),
```

Comment:
I suggest we use client.application for distinguishing applications, leaving client.os.name as `NDTjs`. 

e.g. on measure-app: https://github.com/m-lab/measure-app/blob/master/www/js/measurements/ndt/ndt-browser-client.js#L448

Reply:
@robertodauria addressed this in my latest commit. PTAL?